### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kabir0x23**
## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes

80% of PRs missing newline at the end of the file.
so i add the line in the markdown file, below the code example.: 
> <span style="color: #606c71">Also add an empty line at the end of the file as shown in the above example.</span>

### CODE REVIEW

**Review by AvidCoder101 — APPROVED** [review]
I think that the PR can be merged. @Panquesito7?

File: README.md [hunk 1]
```diff
@@ -41,8 +41,12 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
   "name": "Eddie Jaoude",
   "username": "eddiejaoude"
 }
+ 
 ```
 
+> <span style="color: #606c71">Also add an empty line at the end of the file as shown in the above example.</span>
+
```
Comment (schmelto) [inline README.md:48]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -41,8 +41,12 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
   "name": "Eddie Jaoude",
   "username": "eddiejaoude"
 }
+ 
 ```
 
+> <span style="color: #606c71">Also add an empty line at the end of the file as shown in the above example.</span>
```
Comment (schmelto) [inline README.md:47]:
```suggestion
> Also add an empty line at the end of the file as shown in the example above.
```

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]
Thanks. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

File: README.md [hunk 3]
```diff
@@ -41,8 +41,11 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
   "name": "Eddie Jaoude",
   "username": "eddiejaoude"
 }
+
```
Comment (Panquesito7) [inline README.md:44]: Looks like the prettier is complaining about this line. What should we do about it?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (schmelto) [reply README.md:44]: Tough question can we do how ignore empty lines of n prettier?

Comment (kabir0x23) [reply README.md:44]: What about this only (without changing the above code example)

---
> Also add a new blank line at the end of the code after `}`.

Comment (kabir0x23) [reply README.md:44]: or

---

> Also add a new line at the end of the code after `}` and leave it empty.

Comment (kabir0x23) [reply README.md:44]: last one 

---
> Also add a new line at the end of the code after `}` and leave it empty. Like: 👇
```
1. {
2.   "image": "https://user-images.githubusercontent.com/624760/91057573-48531300-e61f-11ea-9e13-2d7384e42000.png",
3.   "issueId": 66,
4.   "name": "Eddie Jaoude",
5.   "username": "eddiejaoude"
6. }
7.
```

**Review by kabir0x23 — COMMENTED** [review]
.

File: README.md (hunk 3 above)
Comment (kabir0x23) [reply README.md:44]: #### what about these 3 examples (without changing the above JSON code example)

---
1. 
> Also add a new blank line at the end of the code after `}`.
---
2. 
> Also add a new line at the end of the code after `}` and leave it empty.
---
3. 
> Also add a new line at the end of the code after `}` and leave it empty. Like: 👇
```
1. {
2.   "image": "https://user-images.githubusercontent.com/624760/91057573-48531300-e61f-11ea-9e13-2d7384e42000.png",
3.   "issueId": 66,
4.   "name": "Eddie Jaoude",
5.   "username": "eddiejaoude"
6. }
7.
```

**Comment by Pradumnasaraf** [comment]
Closing this PR due the prettier issue.

**Pull request closed without merging**